src/Mod/Assembly/CommandInsertLink.py

```diff
@@ -49,12 +49,12 @@ def __init__(self):
     def GetResources(self):
         return {
             "Pixmap": "Assembly_InsertLink",
-            "MenuText": QT_TRANSLATE_NOOP("Assembly_InsertLink", "Insert Link"),
+            "MenuText": QT_TRANSLATE_NOOP("Assembly_InsertLink", "Insert Component"),
             "Accel": "I",
             "ToolTip": "<p>"
             + QT_TRANSLATE_NOOP(
                 "Assembly_InsertLink",
-                "Insert a Link into the currently active assembly. This will create dynamic links to parts/bodies/primitives/assemblies. To insert external objects, make sure that the file is <b>open in the current session</b>",
+                "Insert a component into the current active assembly. This will create dynamic links to parts, bodies, primitives, and assemblies. To insert external components, make sure that the file is <b>open in the current session</b>",
```

Comment:
```suggestion
                "Insert a component into the active assembly. This will create dynamic links to parts, bodies, primitives, and assemblies. To insert external components, make sure that the file is <b>open in the current session</b>",
```

Was the original "currently" not already grammatically correct? In any case, it might be redundant, so proposing to remove it for a more concise tooltip and to ease translations.